### DISCUSSION TITLE
标注里添加完颜色后在pdf里应用时出现错误

### DISCUSSION THREAD

**Discussion by piliwu** [MuiseDestiny/zotero-style#379] [Q&A] [answered]
PDF.js v2.8.0(build:fdde957)
Message: Localized string not available for pink
Stack:太长了复制不了ㅠㅠ先不写了，大神需要的话再写

**Comment by MuiseDestiny** [comment]
应该贴个截图吧。

**Reply by MuiseDestiny** [reply]
用Beta版本，或者在shift p用官方的颜色名称。你只改颜色就不会报错了。

**Reply by piliwu** [reply]
又有新问题了，我先把上一个问题的截图放上来
<img width="941" alt="0f9f01dbe1977aa3f00de162c4fc9f2" src="https://user-images.githubusercontent.com/89573308/233252874-8cb4c8de-249a-4664-b64c-62ffca5f65f8.png">

**Reply by MuiseDestiny** [reply, marked as answer]
意思是你定义的颜色名称，没有找到对应的中文。这是一个正常报错，因为本身颜色名称就是随便定义的。这个报错一直是有的，beta版本对于这种报错是不会以你截图显示的（会在控制台或者错误日志，这个报错没有什么影响可以正常使用）。所以：1.用beta；2不要改颜色名字，只改颜色值。

**Reply by piliwu** [reply]
新问题是，我在shift+p里设置回了默认颜色
<img width="560" alt="f4a85214f1971a2f496afd90dac2741" src="https://user-images.githubusercontent.com/89573308/233253037-39d95e76-e076-4606-b43c-e9a00bcf4059.png">
但是我在pdf上标注时，还是显示之前添加的颜色，并且报错。报错内容和之前一样。我把插件删掉就恢复了。然而重装插件后还是这样
<img width="278" alt="38bae7ee135874a7e3028b153d89029" src="https://user-images.githubusercontent.com/89573308/233253152-a52c0c74-ae1a-4dd6-9600-12d650827029.png">

**Reply by MuiseDestiny** [reply]
https://www.zotero.org/support/beta_builds

试试beta

**Reply by piliwu** [reply]
5555，大神。我下了beta，现在是这个版本。
<img width="423" alt="42eba1070ce1e6db481386850bfdcb9" src="https://user-images.githubusercontent.com/89573308/233254210-67aad4be-57ea-4c8a-b709-92fc27280b06.png">
但依然报错……
<img width="968" alt="bfcbcf4512baefb02ccb1e1af72c9a3" src="https://user-images.githubusercontent.com/89573308/233254275-6dddc9ac-6e09-4566-898a-f19366ac7c60.png">
而且这个报错会影响使用。当我想删除标注时，一单击右键就报错。删除项不出来。我得每次重启pdf，再单击右键，删除项才能出来。和这个删除项一起出来的还有很气人的默认颜色。在标注时，依然显示添加的颜色，不是默认色。ㅠㅠ
![image](https://user-images.githubusercontent.com/89573308/233254575-564179e8-2ef9-441d-834a-48151952183a.png)

**Reply by MuiseDestiny** [reply]
![image](https://user-images.githubusercontent.com/51939531/233255243-40ef5235-dcfb-4522-9a08-6a0127af4f65.png)

关闭这个功能重启用官方的颜色吧。

**Reply by MuiseDestiny** [reply]
可能每个电脑体质不一样，有的设备就是报错。

**Reply by piliwu** [reply]
好的，把这个关了就恢复了。55555我还挺想加颜色的

**Reply by piliwu** [reply]
哭泣，谢谢大神

**Reply by ZD802** [reply]
![image](https://github.com/user-attachments/assets/b8dcc34a-4109-4cd0-b376-52c933b109a6)
你好，这个无法点应用是为什么呀

**Comment by Nalakuvvara** [comment]
你好，自己设置的颜色现在不可以使用了，是什么原因呢？自己在pdf中打开还是初始颜色，如何设置呢？插件是5.2.0，自动更新后就这样了
![Uploading 截屏2024-11-04 21.50.39.png…]()
![Uploading 截屏2024-11-04 21.51.00.png…]()

**Reply by Nalakuvvara** [reply]
![Uploading 截屏2024-11-04 21.50.39.png…]()
